elyra/pipeline/kfp/processor_kfp.py

```diff
@@ -495,7 +497,32 @@ def _cc_pipeline(
 
                 operation_artifact_archive = self._get_dependency_archive_name(operation)
 
-                self.log.debug(f"Creating pipeline component:\n {operation} archive : {operation_artifact_archive}")
+                self.log.debug(
+                    f"Creating pipeline component archive '{operation_artifact_archive}' for operation '{operation}'"
+                )
+
+                volume_mounts = operation.component_params.get(MOUNTED_VOLUMES)
```

Comment:
This seems a little odd to assign `volume_mounts` then, if non-None, re-initialize it.  Perhaps just setting it `None` with some type hinting might make it more clear?
```suggestion
                volume_mounts: Optional[Dict[str,str]] = None
```